scripts/launch_validator_node.sh

```diff
@@ -83,9 +83,17 @@ function become_a_validator {
 
 function launch_nodes {
   define_config_file $NETWORK
+
   start_validator_node
-  echo "NODE_NAME=$NODE_NAME" >> configs/${CONFIG_FILE}
+
+  sed -i "s|NODE_NAME=.*|NODE_NAME=$NODE_NAME|" configs/${CONFIG_FILE}
```

Comment:
This line should be before `start_validator_node` one.

Reply:
done